tools/run_tests/python_utils/upload_rbe_results.py

```diff
@@ -189,8 +216,9 @@ def _get_resultstore_data(api_key, invocation_id):
         elif 'tests' not in action['testAction']['testSuite']:
             continue
         else:
-            test_cases = action['testAction']['testSuite']['tests'][0][
```

Comment:
not that surprisingly, it turns out that some  'tests' elements have multiple groups of tests. The original version of the code just skips them. 
This only actually happens for a few tests so the overall quality of the old bigquery data is still pretty good.